### DISCUSSION TITLE
Interpretating results

### DISCUSSION THREAD

**Discussion by anibalbezerra** [dgehringer/sqsgenerator#53] [General]
After a few runs of your code (really easy, btw!), I have a general question regarding comparing your method with the traditional Zhunger's one. The annealing in the last tends to generate structures that do not preserve the crystal symmetry. Conversely, the SQSGENERATOR seems to exchange (or choose, if preferable) the atoms' positions but keeps the original structure. In your manuscript, it is mentioned that the annealing method would lead to smaller random alloys. I know that it should be a difficult answer, but how big should be the structure generated by SQSGENERATOR to ensure the required randomness? What property should we converge to be sure the generated structure is the "best one"? Even the Zhunger SQS is still a doubt for me, especially since it changes the symmetry. Would the crystal symmetry be an important aspect of the SQS? Namely, in Zhunger's perspective, the SQS is a structure with different symmetry, but the correlations among neighbors are accounted for to ensure its response relates to the experimental one.

**Comment by dhoulek** [comment]
Hello @anibalbezerra , thanks for your feedback :-)

I am unsure what type of annealing you refer to (by the way, Zhunger is probably misspelled as Alex Zunger).
Our approach is entirely geometrical, so no annealing is possible. The underlying lattice, of course, remains the same. 

The idea of SQS is to generate "as small as possible representative cell". So, regarding the convergence, you have to define a measure of representativeness for you. Note that if this is a physical property, the sqsgenerator can hardly give you an answer (because it has no idea about the actual material). If you define it via some geometrical measures (e.g. you manage to quantify how a distribution (=histogram) of your local environments differs from an ideal one), you can probably post-process the sqsgenerator results (or contribute an add-on :-)

Suppose your measure is, for example, total energy. In that case, it is of the utmost importance to ensure that the side effects related to the SQS size are not confused with numerical inaccuracies stemming from different cells (i.e., different k-meshes in DFT, etc.).

In the past, we also used symmetry to measure how "good" an SQS is for tensorial properties. Maybe you want to check [this publication (DOI: 10.1103/PhysRevB.90.184106)](https://journals.aps.org/prb/abstract/10.1103/PhysRevB.90.184106).

_David_

**Reply by anibalbezerra** [reply]
Thanks for the response. I'll take a read to the manuscript you referenced. But I think I got your point.
